components/Layout/NavMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useState } from "react";
+import {
+  Box,
+  useColorModeValue,
+  NAMED_COLORS,
+  Portal,
+  useConst,
+} from "@ironfish/ui-kit";
+
+import NavListOfLinks from "./NavListOfLinks";
+
+interface NavMenuProps {
+  menuRef: RefObject<FC | null>;
+  children: ReactNode;
+}
+
+const NavMenu: FC<NavMenuProps> = ({ menuRef, children }) => {
+  const colors = useColorModeValue(
+    { bg: NAMED_COLORS.DEEP_BLUE },
+    { bg: NAMED_COLORS.WHITE }
+  );
```

Comment:
May be better to use something like this?
```suggestion
  const background = useColorModeValue(NAMED_COLORS.DEEP_BLUE, NAMED_COLORS.WHITE);
```